applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -28,6 +28,20 @@ config BLE_LE_POWER_CONTROL_ENABLED
 	  dynamically and automatically during connection, which provides effective
 	  communication.
 
+config BT_PREFERRED_MIN_PRES_DLY_US
```

Unsure if we should create `BT_*` configs. Suggestions are welcome.

---

Should be BT_AUDIO, in line with the other configs we have, and they should probably be lower in this file, so that they are all close together. Also suggest:
```suggestion
config BT_AUDIO_PREF_PRES_DLY_MIN_US
```

---

Will update